hack/test-e2e-provider-local.sh

```diff
@@ -21,24 +21,26 @@ source "$repo_root/gardener/hack/ci-common.sh"
 
 echo '172.18.255.1 api.ping-test.local.external.local.gardener.cloud' >> /etc/hosts
 echo '172.18.255.1 api.con-test.local.external.local.gardener.cloud' >> /etc/hosts
+echo '172.18.255.1 api.enc-con-test.local.external.local.gardener.cloud' >> /etc/hosts
 echo '172.18.255.1 api.e2e-force-del.local.external.local.gardener.cloud' >> /etc/hosts
 echo '127.0.0.1 garden.local.gardener.cloud' >> /etc/hosts
 
-make kind-up
+make kind-ha-single-zone-up
```

Comment:
Why do you want to test the HA case?

Reply:
The HA case is to test a two nodes setup, since only the communication between the nodes is encrypted, a single node setup will not be affected by this change at all.

Reply:
Ah, no, the number of nodes in the shoot is not related to this setting. It is set [here](https://github.com/gardener/gardener-extension-networking-cilium/blob/a876dde3ef97194ed00fa12eede361310efccbd8/test/e2e/common_test.go#L83-L84). We always test with two nodes.
ha-single-zone actually tests with an [HA control-plane](https://github.com/gardener/gardener/blob/master/docs/usage/high-availability/shoot_high_availability.md).

Reply:
🙈 i'll roll back the `ha-single-zone` change